app/services/password_hasher.rb

```diff
@@ -0,0 +1,24 @@
+class PasswordHasher
+  # IMPORTANT! changing COST will invalidate existing password hashes.
+  COST = if Rails.env.test?
+           '800$8$1$'.freeze # SCrypt::Engine.calibrate(max_time: 0.01)
```

Comment:
could we save these magical-ish strings as separate constants? Side benefit might be that the constant names would replace need for comments